dts/arm/realtek/ec/rts5912.dtsi

```diff
@@ -56,6 +56,21 @@
 			#clock-cells = <2>;
 		};
 
+		slwtmr0: slwtmr0@4000c200 {
+			compatible = "realtek,rts5912-slwtmr0";
+			reg = <0x4000c200 0x10>;
+			interrupt-parent = <&nvic>;
+			interrupts = <202 0>;
```

Comment:
These properties are not used.

Reply:
These properties will be necessary in the new yaml file of slow timer